sys/net/gnrc/link_layer/lwmac/lwmac.c

```diff
@@ -293,14 +293,15 @@ void lwmac_set_state(gnrc_netif_t *netif, gnrc_lwmac_state_t newstate)
                 uint32_t alarm;
 
                 rtt_clear_alarm();
-                alarm = random_uint32_range(RTT_US_TO_TICKS((3 * GNRC_LWMAC_WAKEUP_DURATION_US / 2)),
-                                            RTT_US_TO_TICKS(GNRC_LWMAC_WAKEUP_INTERVAL_US -
-                                                            (3 * GNRC_LWMAC_WAKEUP_DURATION_US / 2)));
+                alarm = random_uint32_range(RTT_US_TO_TICKS((3 *
```

Comment:
Please keep the original indentation and wrapping when possible. Also drop unnecessary `\`.

Reply:
Yes. This was an old commit. Will follow the orignal coding style from now on.